packages/core/src/prompts/prompt.ts

```diff
@@ -162,6 +162,9 @@ export default class Prompt {
 		if (char && (char.toLowerCase() === 'y' || char.toLowerCase() === 'n')) {
 			this.emit('confirm', char.toLowerCase() === 'y');
 		}
+		if (char) {
+			this.emit('key', char.toLowerCase());
```

Comment:
I doubt if there's any specific use case where user wants to distinguish the cases

Reply:
Oh I see that's the behavior we've already had. Then let's not bother it.